src/apps/somm/shortcuts.e2e.ts

```diff
@@ -35,4 +35,30 @@ describe('getShortcutDefinitions', () => {
       expect(transactions.length).toEqual(2)
     })
   })
+
+  describe('withdraw.onTrigger', () => {
+    it('should return transactions', async () => {
+      const shortcuts = await hook.getShortcutDefinitions(
+        NetworkId['arbitrum-one'],
+      )
+      const shortcut = shortcuts.find((shortcut) => shortcut.id === 'withdraw')
+      expect(shortcut).toBeDefined()
+      expect(shortcuts[1].id).toBe('withdraw')
```

Comment:
do we need to check that `withdraw` is exactly 2nd?
looks like we checked above that at least one shortcut with an id = `withdraw` exists